src/microbe_stage/MicrobeHUD.cs

```diff
@@ -242,7 +242,9 @@ public class MicrobeHUD : Control
     private AnimationPlayer patchOverlayAnimator = null!;
     private TextureButton editorButton = null!;
     private CustomDialog? extinctionBox;
+    private PatchExtinctionBox? patchExtinctionBox;
     private CustomDialog? winBox;
+    private PackedScene patchExtinctionBoxScene = null!;
```

Comment:
Why aren't these two newly added very closely related variables next to each other?